build.gradle

```diff
@@ -255,6 +255,7 @@ configure(javaProjects) { subproject ->
 		configurations.all {
 			resolutionStrategy.cacheChangingModulesFor 60, 'minutes'
 			exclude group: 'ch.qos.logback', module: 'logback-classic'
+			exclude group: 'org.projectreactor', module: 'reactor-core'
```

Comment:
Side note, is this not possible to have a 4.1.4 with optional dependency on reactor already and document that. It doesn't seem to be breaking any functionality but the arcanes of maintaining popular libraries is a difficulty to overcome you have the experience with :)